datalad/distribution/publish.py

```diff
@@ -49,16 +51,17 @@ def _log_push_info(pi_list):
 
 
 class Publish(Interface):
-    """Publish a dataset (e.g. to a web server)
+    """Publish a dataset to a known :term:`sibling`.
 
-    This makes the current state of a dataset available to a sibling of the
-    dataset. That sibling needs to be known to the dataset before.
-    `create_publication_target` commands are meant to set up such a sibling, but
-    generally you can publish to any sibling added to a dataset.
-    Publishing may or may not include subdatasets.
-    By default file handles are published without their actual content,
-    but they can be published including the content, of course.
+    This makes the last saved state of a dataset available to a sibling
+    or special remote data store of the dataset which must already exist
```

Comment:
Oh... Shouldn't we now introduce some new term to improve clarity, e.g. 'bastard', when taking about special remotes? :-)